client/src/components/Carousel.jsx

```diff
@@ -3,13 +3,13 @@ import CarouselImage from './CarouselImage.jsx';
 import Arrow from './Arrow.jsx';
 
 const Carousel = React.forwardRef((props, ref) => {
-  // console.log(props.mainPhotos);
+  // console.log(props);
   return(
     <div className="carousel-container">
       <Arrow direction="left" glyph="&#10094;" clickFunction={props.carouselLeftArrowClick} />
       <div className="carousel-wrapper" ref={ref}>
         {props.photos.map((photo, index) => {
-          return <CarouselImage key={index} photo={photo} />
+          return <CarouselImage key={index} id={index} photo={photo} handleCarouselPictureClick={props.handleCarouselPictureClick} />
```

Comment:
Are your photos hardcoded or from the database? If it is from the database, it is better to use the "id" from MySQL instead of map index.

Reply:
Yah it's hard coded right now. Good point, I will use "id" from MySQL when getting data from the database. Thanks!